src/pages/ConfigurePatch.tsx

```diff
@@ -13,13 +15,31 @@ export const ConfigurePatch: React.FC = () => {
   });
   return (
     <PageWrapper>
-      {loading ? (
-        <PatchAndTaskFullPageLoad />
-      ) : error ? (
-        <div>{error.message}</div>
-      ) : (
-        <ConfigurePatchCore patch={data.patch} />
-      )}
+      {(function() {
+        if (loading) {
+          return <PatchAndTaskFullPageLoad />;
+        } else if (error) {
+          return (
+            <PageLayout>
+              <div>{error.message}</div>
+            </PageLayout>
+          );
+        }
+        const variants = get(data, "patch.project.variants");
```

Comment:
same comment as above move this outside of the return  and do this logic outside of the jsx.